src/App.tsx

```diff
@@ -16,16 +28,18 @@ export const App: React.FC = () => {
 
             <div className="control">
               <input
-                type="text"
+                type="search"
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={((event) => changedQuery(event.target.value))}
```

Comment:
one pair of parenthesis is extra here
```suggestion
                onChange={event => changedQuery(event.target.value)}
```